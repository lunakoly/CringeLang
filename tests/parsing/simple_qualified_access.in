fun doThings(): Test.Rest
    a.name, b.index = "test", 20