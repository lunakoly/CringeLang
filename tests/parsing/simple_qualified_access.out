==== Raw AST ====
[fun doThings ([]): Test.Rest [[([a.name, b.index] = ["test", 20])]]]

==== Resolved AST ====
[fun doThings ([]): [ERROR:DIFFICULT_TYPE] [[([a.name, b.index] = ["test", 20])]]]

==== Diagnostics ====

==== Done ====
